test/common/common/utility_fuzz_test.cc

```diff
@@ -0,0 +1,31 @@
+#include "common/common/utility.h"
+
+#include "test/fuzz/fuzz_runner.h"
+
+#include "absl/strings/string_view.h"
+
+namespace Envoy {
+namespace Fuzz {
+
+DEFINE_FUZZER(const uint8_t* buf, size_t len) {
+
+  uint64_t out;
+  std::string stringBuffer(*buf, len);
+  absl::string_view abslString = stringBuffer;
```

Comment:
This shouldn't be needed, it will be implicitly converted to a string view as needed AFAICT.